index.ts

```diff
@@ -38,7 +43,7 @@ export async function main() {
     console.log('Configuring authority')
     const balance = await connection.getBalance(authority.publicKey)
     if (balance < 100) {
-        const signature = await connection.requestAirdrop(authority.publicKey, LAMPORTS_PER_SOL * 1000);
+        const signature = await connection.requestAirdrop(authority.publicKey, LAMPORTS_PER_SOL * 200);
```

Comment:
I think airdrop will work only for private network, what about adding it as a parameter? like `process.env.AIRDROP_EXIST` or something